test-common/uk/gov/hmrc/alcoholdutyaccount/common/TestData.scala

```diff
@@ -224,13 +231,336 @@ trait TestData extends ModelGenerators {
+          subTransaction = "6132",
+          outstandingAmount = Some(BigDecimal("5000")),
+          items = Seq(
+            FinancialTransactionItem(
+              subItem = "000",
+              dueDate = Some(ReturnPeriod.fromPeriodKeyOrThrow(periodKey).dueDate()),
+              amount = BigDecimal("5000")
+            ),
+            FinancialTransactionItem(
+              subItem = "001",
+              dueDate = Some(ReturnPeriod.fromPeriodKeyOrThrow(periodKey).dueDate()),
+              amount = BigDecimal("4000")
+            )
+          )
+        ),
+        FinancialTransaction(
+          sapDocumentNumber = sapDocumentNumberGen.sample.get,
+          periodKey = Some(periodKey),
+          chargeReference = Some(chargeReferenceGen.sample.get),
+          originalAmount = BigDecimal("2000"),
+          mainTransaction = TransactionType.toMainTransactionType(TransactionType.Return),
+          subTransaction = "6132",
+          outstandingAmount = Some(BigDecimal("2000")),
+          items = Seq(
+            FinancialTransactionItem(
+              subItem = "000",
+              dueDate = Some(ReturnPeriod.fromPeriodKeyOrThrow(periodKey).dueDate()),
+              amount = BigDecimal("2000")
+            )
+          )
+        )
+      )
+    )
+
+  val singleFullyUnallocatedPayment: FinancialTransactionDocument =
+    FinancialTransactionDocument(
+      financialTransactions = Seq(
+        FinancialTransaction(
+          sapDocumentNumber = sapDocumentNumberGen.sample.get,
+          periodKey = None,
+          chargeReference = None,
+          originalAmount = BigDecimal("-9000"),
+          mainTransaction = TransactionType.toMainTransactionType(TransactionType.PaymentOnAccount),
+          subTransaction = "6132",
+          outstandingAmount = Some(BigDecimal("-9000")),
+          items = Seq(
+            FinancialTransactionItem(
+              subItem = "000",
+              dueDate = Some(ReturnPeriod.fromPeriodKeyOrThrow(periodKey).periodFromDate()),
+              amount = BigDecimal("-9000")
+            )
+          )
+        )
+      )
+    )
+
+  val singlePartiallyAllocatedPayment: FinancialTransactionDocument =
+    FinancialTransactionDocument(
+      financialTransactions = Seq(
+        FinancialTransaction(
+          sapDocumentNumber = sapDocumentNumberGen.sample.get,
+          periodKey = None,
+          chargeReference = None,
+          originalAmount = BigDecimal("-9000"),
+          mainTransaction = TransactionType.toMainTransactionType(TransactionType.PaymentOnAccount),
+          subTransaction = "6132",
+          outstandingAmount = Some(BigDecimal("-5000")),
+          items = Seq(
+            FinancialTransactionItem(
+              subItem = "000",
+              dueDate = Some(ReturnPeriod.fromPeriodKeyOrThrow(periodKey).dueDate()),
+              amount = BigDecimal("-5000")
+            ),
+            FinancialTransactionItem(
```

Comment:
This isn't how it works, POA replaces, so no 001 and original = outstanding = amount = -5000